.github/workflows/build.yaml

```diff
@@ -0,0 +1,34 @@
+# This is a GitHub workflow defining a set of jobs with a set of steps.
+# ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
+#
+name: Build
```

Comment:
We build as a test, and the file has a todo about "Check VNC desktop works", so maybe it should be named "test.yaml" instead?

In other jupyterhub org projects, we do a basic "check build" as part of a release.yaml workflow, but this leans to become a test workflow it seems.

Reply:
Done!